mock-api/getApiResult.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import update from 'react-addons-update';
+import {
+  getChartTypeDefaultOpts,
+  getChartTypeObject,
+} from '../app/utils/chartTypeUtils';
+import { parseRawData, transformParsedData } from '../app/utils/rawDataHelpers';
+import Baby from 'babyparse';
```

Comment:
I'm curious on how you decided upon babyparse, vs other csv parsers like node-csv or csv-parser